src/controllers/comment.controller.js

```diff
@@ -24,6 +24,22 @@ class CommentController {
     ResponseService.setSuccess(201, 'Your comment was submitted successfully', commentInformation);
     return ResponseService.send(res);
   }
+
+
+  /**
+   *
+   *
+   * @static
+   * @param {req} req
+   * @param {req} res
+   * @memberof CommentController
+   * @returns {object} this function deletes comment on posted on thread
+   */
+  static async deleteComment(req, res) {
+    await CommentService.deleteCommentById({ id: req.params.commentId });
+    ResponseService.setSuccess(200, 'Comment was deleted');
```

Comment:
This message is not so much descriptive. you can say `comment deleted successfully` instead of `comment was deleted` which might indicate that it was deleted earlier not as the user performs the action.